src/main/resources/application.yml

```diff
@@ -25,7 +25,9 @@ server:
 # Spring
 ###############################################################################
 spring:
-  application.name: "myhome-service"
+  application:
+    name: "myhome-service"
+    version: 1.2.0-release
```

Comment:
Let's not include this in the banner. As this gets outdated very soon, also we should take care of updating this after every release.
I am wondering if there is any way to automatically get it from Gradle configuration.